### ISSUE TITLE
My BCH are found but I cant transfer them to a BCH adress

### ISSUE THREAD

**Issue by jockeo1** [bitpay/copay-recovery#79]
Hello!

I had bitcoins at my copay wallet and I wanted to retrieve my bitcoin cash and followed the guide @  https://blog.bitpay.com/bitcoin-cash-recovery/ 

My BCHs are found after I put in my recovery phrase and scanned and everything is good but when i press the transfer button (after I filled in a BCH adress) i get the message **"Please enter a valid adress"**?

I tried to send to different BCH adresses both at bittrex and at bitfinex. 

Any1 else who have had this problem? what am I doing wrong :-/?

**Comment by nicaport** [comment]
Yes, I had this problem yesterday.  They are using a new address format and are telling everyone to use their address translator tool. But I used it, and then input the translated address into the recovery tool. It said the funds were successfully sent, but then the funds never arrived! My 5k has now vanished into thin air, and the BitPay team has not yet responded to me. When I voiced my concern to them over this on github, they closed the issue even though it was unresolved. Really not sure what's going on at BitPay... doesn't seem great.

**Comment by nicaport** [comment]
Do NOT use this tool: https://bitpay.github.io/address-translator/.

**Comment by nicaport** [comment]
Still no response from the bitpay team? Copay is my preferred wallet on iPhone. I recommend it to everyone so this support is very disappointing. @JDonadio @matiu

**Comment by GOF76** [comment]
Hi there... i had the same problem, after i was recover the BCH and filled in the BCH Adress i get the mesage "please enter valid adress"??
can we please get any response from bitpay whats goin wrong?!

**Comment by JDonadio** [comment]
Hi guys, please take a look there: https://github.com/bitpay/copay-recovery/issues/75#issuecomment-332187010

**Comment by GOF76** [comment]
hey JDonadio, i solved the problem i just updatet the wallet Version, now i can claim it directly in the bitpay wallet... thanks
